### PR TITLE
test: migration of application test to Jest

### PR DESCRIPTION
**Pull request by ctschacher**
Migrated Cypress test to cover basic management application functions (create, delete)

gravitee-io/issues#7668

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/test-7668-application-tests-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-rrtmarxztw.chromatic.com)

### CODE REVIEW

File: gravitee-apim-e2e/api-test/applications/applications.spec.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        applicationResource.createApplicationRaw({
+          orgId,
+          envId,
+          newApplicationEntity: ApplicationsFaker.newApplication(),
+        }),
+      );
+      expect(application.status).toEqual('ACTIVE');
+    });
+
+    test('should delete an application', async () => {
```
Comment (Okhelifi) [inline gravitee-apim-e2e/api-test/applications/applications.spec.ts]: Is it a `test` or an `afterEach` 🤔  Imho it's more an afterEach as your not testing the application deletion in this describe.

File: gravitee-apim-e2e/api-test/applications/applications.spec.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          orgId,
+          envId,
+          application: application.id,
+        }),
+      );
+    });
+  });
+});
+
+describe('Application tests with unauthorized users', () => {
```
Comment (Okhelifi) [inline gravitee-apim-e2e/api-test/applications/applications.spec.ts]: shouldn't we add an `afterAll` to clean the db at the end ?

Comment (ctschacher) [reply gravitee-apim-e2e/api-test/applications/applications.spec.ts]: Yes right, that's indeed missing here, I'll fix that.

File: gravitee-apim-e2e/api-test/applications/applications.spec.ts (hunk 1 above)
Comment (ctschacher) [reply gravitee-apim-e2e/api-test/applications/applications.spec.ts]: Well, actually, I do want to test both, the create & the deletion within that describe block.

**Review by Okhelifi — APPROVED** [review]

**Pull request merged**